samples/SocialWeather/project.json

```diff
@@ -17,7 +17,7 @@
     "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
   },
   "frameworks": {
-    "netcoreapp1.1": {}
+    "netcoreapp1.2": { }
```

Comment:
Here, we can downgrade, but in other projects (ChatSample) some of the packages are pulling the NetCore.App package up to 1.2. It seems odd to have the package 1.2 and the TFM `netcoreapp1.1` so I just updated all to be consistent.